apps/agent/src/main/window-manager.ts

```
@@ -0,0 +1,74 @@
+import { resolveHtmlPath } from './util';
+import * as path from 'path';
+
+class AppWindow {
+	aboutWindow: BrowserWindow | null;
+	splashScreenWindow: SplashScreen;
+	setupWindow: BrowserWindow;
+	rootPath: string;
+	private static instance: AppWindow;
+	constructor(rootPath: string) {
+		if (AppWindow.instance) {
+			return AppWindow.instance;
```

Biome (1.9.4)

[error] The constructor should not return a value.

The constructor is here:

Returning a value from a constructor is ignored.

(lint/correctness/noConstructorReturn)